corehq/apps/data_analytics/tasks.py

```diff
@@ -0,0 +1,36 @@
+from celery.schedules import crontab
+from celery.task import periodic_task
+from celery.utils.log import get_task_logger
+import datetime
+
+from corehq.apps.data_analytics.malt_generator import MALTTableGenerator
+from dimagi.utils.dates import DateSpan
+from dimagi.utils.django.email import send_HTML_email
+
+
+logger = get_task_logger(__name__)
```

Comment:
another option might be to just save all error messages and send one email at the end